### DISCUSSION TITLE
Organization member not triggering email notifications in PR

### DISCUSSION THREAD

**Discussion by afleite93** [community/community#23500] [New to GitHub]
<p>Hi,</p>
<p>Currently in my organization we have a member that does not trigger email notifications when assigning reviewers in pull requests, it only triggers web notifications and the reviewers only receive the web notifications. When other organization member in the same pull request assigns the reviewers both web and email notifications are triggered and the reviewers receive both.</p>
<p>Can you help me with this problem?</p>

**Comment by queenofcorgis** [comment]
<p>Hi <a class="mention" href="/afleite-centi">@afleite-centi</a> :wave:</p>
<p>Thanks for posting this - I am working with our Notifications team to see if we can find a solution or the cause of this. We may have some follow up questions for you as we dig in!</p>

**Comment by queenofcorgis** [comment]
<p>Wanted to update you and anyone watching this topic that we’re still investigating, but we have eyes :eyes: on this! :stars:</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there - have you checked your spam or junk filters - our records show emails are sending!</p>

**Comment by afleite93** [comment]
<p>Hi,</p>
<p>That’s strange, the only email that I received regarding the CENTITVC/FDPanel PR <span class="hashtag">#3</span> was from <span class="mention">@vmachado-centi</span>. There’s nothing in the spam/junk files.  I will check with my colleague tomorrow.</p>
<p>I will create a test repository for this in order to be more simple in your side to check it because all email notifications from <span class="mention">@rfigueiredo-centi</span> are not being received at all.  I will also check with the IT department about this.</p>
<p>Thank you for your help.</p>

**Comment by afleite93** [comment]
<p>Hi there,</p>
<p>So, the problem was on the Office365, we added the <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> server to the Trusted Servers lists and we also added it to the Spam Software allow list.</p>
<p>Thank you for your help. :slight_smile:</p>

**Comment by queenofcorgis** [comment]
<p>Hooray! Thanks <a class="mention" href="/afleite-centi">@afleite-centi</a> for the update :tada:  I’m so glad we got to the bottom of this :heart: Appreciate your patience!</p>
